sesman/sesman.ini.in

```diff
@@ -153,6 +153,17 @@ FileUmask=077
 ; not use this setting for GNOME 4, or other file managers
 #UseNautilus3FlistFormat=true
 
+; sound redirection
+; workaround for Microsoft mstsc.exe to suppress noise.
+; SoundNumSilentFramesAAC | SoundNumSilentFramesMP3 silent frames are sent before SNDC_CLOSE is sent.
+; during SoundMsecDoNotSend mS after SNDC_CLOSE is sent, sound data is not send.
+; depending on the environment, it might be necessary to increase values.
+; Defaults: SoundNumSilentFramesAAC=4, SoundNumSilentFramesMP3=2, SoundMsecDoNotSend=1000
+; If commented or set to 0, this workaround is not applied.
+SoundNumSilentFramesAAC=4
+SoundNumSilentFramesMP3=2
+SoundMsecDoNotSend=1000
+
 [ChansrvLogging]
```

Comment:
Suggest you define these as follows:-

```
#SoundNumSilentFramesAAC=4
#SoundNumSilentFramesMP3=2
#SoundMsecDoNotSend=1000
```

It's a small point. It means we're setting as little as possible by default in the config file. If for some reason we then need to change after delivery we'll be affecting less users.

Reply:
Understood.
I commented out parameters.
After that I tried to squash and push though, did I do something wrong?